src/code42cli/cmds/securitydata.py

```diff
@@ -15,8 +15,8 @@
 from code42cli.options import incompatible_with
 from code42cli.options import OrderedGroup
 from code42cli.options import sdk_options
-from code42cli.util import find_format_width
-from code42cli.util import format_to_table
+from code42cli.output_formats import format_option as format_output
```

Comment:
Why rename the import here?

Reply:
Another option exists here with same name, used in `search` subcommand. It doesn't support same formatting options as of now.
I am planning to touch those in a separate branch as I need more information on what fields should be considered to be displayed in the output for such commands.